articles/tutorials/calling-an-external-idp-api.md

```diff
@@ -51,15 +51,22 @@ The following code executes a client credentials exchange so that you receive an
 
 ```har
 {
-  "method": "POST",
-  "url": "https://${account.namespace}/oauth/token",
-  "headers": [
-    { "name": "Content-Type", "value": "application/json" }
-  ],
-  "postData":{
-    "mimeType": "application/json",
-    "text": "{ \"client_id\": \"${account.clientId}\", \"client_secret\": \"${account.clientSecret}\", \"audience\": \"https://${account.namespace}/api/v2/\", \"grant_type\": \"client_credentials\" }"
-  }
+	"method": "POST",
```

Comment:
@khorne3 I'm pretty sure this is wrong, and the way it was before was the correct one. The valid way to get an APIv2 token is described here https://auth0.com/docs/api/management/v2/tokens#1-get-a-token

I even tried to get a token with the new way described here but if I don't specify the `audience` param, the token I'm getting is an opaque token, not a JWT, and if I try to use it at the Management APIv2 I get the following error:

![image](https://cloud.githubusercontent.com/assets/11715799/23702243/1617fe0c-0403-11e7-93c4-fa45f8665f4c.png)

Did someone from engineering give this input or can I fix it and merge?